### PR TITLE
Update docs for RFC 176 syntax and canonical URL

### PR DESCRIPTION
**Pull request by williamhaley**
#15723 Update `ember-debug` package to use rfc176 imports/naming conventions and update doc URLs to use `../release/..` for canonical path

### CODE REVIEW

File: packages/ember-debug/tests/main_test.js [hunk 1]
```diff
@@ -55,27 +55,27 @@ moduleFor('ember-debug', class extends TestCase {
     ENV._ENABLE_DEPRECATION_OPTIONS_SUPPORT = originalDeprecationOptions;
   }
 
-  ['@test Ember.deprecate does not throw if RAISE_ON_DEPRECATION is false'](assert) {
+  ['@test deprecate does not throw if RAISE_ON_DEPRECATION is false'](assert) {
```
Comment (williamhaley) [inline packages/ember-debug/tests/main_test.js]: Updating names in tests wasn't in the scope original issue, but I figured I could update them while I was in there. I'm happy to revert this though.

Comment (toddjordan) [reply packages/ember-debug/tests/main_test.js]: its what I was trying to say here, so yes, you are on the right track:

> Additionally we'd like to convert references to old Ember namespaced classes to their simplified post RFC 176 names.
> (example: documentation might refer to Ember.Object, while we'd want it to be EmberObject now)
> See the readme for https://github.com/ember-cli/ember-rfc176-data for the mappings.

**Comment by toddjordan** [comment]
This looks great.  Thanks, @williamhaley !

**Comment by toddjordan** [comment]
@williamhaley Could you reword the commit prepend [doc beta], so we can get this into 3.0?
See https://github.com/emberjs/ember.js/blob/master/CONTRIBUTING.md#documentation
(I'll also update the issue)

**Comment by williamhaley** [comment]
@toddjordan commit tag updated 👍

**Pull request merged**